subprojects/plugin-development/src/main/java/org/gradle/plugin/devel/tasks/GeneratePluginDescriptors.java

```diff
@@ -58,11 +58,21 @@ public GeneratePluginDescriptors() {
         );
     }
 
-    @Internal("The relevant data is tracked via implementationClassById")
+    /**
+     * Container for all plugin declarations declarations.
+     *
+     * @see #getImplementationClassById()
```

Comment:
I'd remove this line completely, since the Javadoc shouldn't point to an internal method. And the `@Internal` annotation already references it.

Reply:
Fair enough, corrected